Source/WebKit/Scripts/process-entitlements.sh

```diff
@@ -444,7 +444,17 @@ fi
 
 function ios_family_process_model_entitlements()
 {
-    ios_family_process_webcontent_shared_entitlements
+    plistbuddy Add :com.apple.QuartzCore.secure-mode bool YES
+    plistbuddy Add :com.apple.QuartzCore.webkit-end-points bool YES
+    plistbuddy add :com.apple.QuartzCore.webkit-limited-types bool YES
+    plistbuddy Add :com.apple.private.memorystatus bool YES
+    plistbuddy Add :com.apple.private.webkit.use-xpc-endpoint bool YES
```

Comment:
The XPC endpoint entitlement may not be needed, unless you intend to send the Launch Services database to the new process.

Reply:
I don't intend so! Will remove it.